src/content_cao.cpp

```diff
@@ -1850,5 +1850,10 @@ std::string GenericCAO::debugInfoText()
 	return os.str();
 }
 
+std::string GenericCAO::infoText()
```

Comment:
no, set this definition to content_cao.h directly to permit inlining implictly by compiler.